docs/packages/xstate-svelte/index.md

```diff
@@ -91,7 +91,7 @@ This special `useMachine` hook is imported from `@xstate/svelte/lib/fsm`
 
 ```svelte
 <script>
-  import { useMachine } from '@xstate/svelte/lib/fsm';
+  import { useMachine } from '@xstate/svelte/lib/fsm.js';
```

Comment:
Without the `.js` extension there is an error.

Reply:
Will create separate PR for docs.

Reply:
Hint for the docs: The need for including the file extension comes from native es-module-loading:
https://nodejs.org/api/esm.html#esm_mandatory_file_extensions